### PR TITLE
v0.3.10: 

### PR DESCRIPTION
**Pull request by andywerner**
introducing modes (incl CALIBRATION_TOOL), now checking with MrbPlugin if modes are enabled.

### CODE REVIEW

File: octoprint_findmymrbeam/__init__.py [hunk 1]
```diff
@@ -318,7 +323,7 @@ def is_enabled(self):
 	def _not_disabled(self):
 		enabled = False
 		try:
-			if self._support_mode:
+			if len(self._get_internal_modes()) > 0:
```
Comment (amrsoll) [inline octoprint_findmymrbeam/__init__.py:326]: `len` is unnecessary. could just be `if self._get_internal_modes():` as empty lists are False-ish

File: octoprint_findmymrbeam/__init__.py [hunk 2]
```diff
@@ -406,7 +430,7 @@ def _perform_update_request(self, uuid, scheme, port, path, search_id=None, http
 					"public_ip6: %s, hostname: %s, local_ips: %s, netconnectd_state: %s, support_mode: %s",
 					(ip4_status_code if ip4_status_code > 0 else ip4_err), self._public_ip,
 					(ip6_status_code if ip6_status_code > 0 else ip6_err), self._public_ip6,
-					hostname, ", ".join(local_ips), netconnectd_state, self._support_mode)
+					hostname, ", ".join(local_ips), netconnectd_state, support_mode)
```
Comment (amrsoll) [inline octoprint_findmymrbeam/__init__.py]: Don't we want to show the calibration mode?

**Review by amrsoll — APPROVED** [review]

File: octoprint_findmymrbeam/__init__.py (hunk 2 above)
Comment (amrsoll) [reply octoprint_findmymrbeam/__init__.py]: ah nvm this is not the server side

File: octoprint_findmymrbeam/__init__.py (hunk 1 above)
Comment (andywerner) [reply octoprint_findmymrbeam/__init__.py:326]: yes. and I wanted to make it explicit since it's not so obvious here that the return val is a list

**Pull request merged**